policy/policy.sample.json

```diff
@@ -0,0 +1,71 @@
+{
+    "SecurityPolicies": [
+        {
+            "Name": "policy1",
+            "PolicyID": "3710940D-FBC0-4A44-A95D-E549E628B4DF", # unique identifier across tenants, auto generated for POST.
+            "AppliedTo": [ # can attach multiple tenants to which the policy can be applied to.
+                {
+                    "Tenant":"tenant1", #optional if TenantID below is provided 
+                    "TenantID": "3A6E467F-370B-43EA-BCEC-C28912EDD752", # optional but good to have to relate things as names are not unique
+                    "Segment": "Segment1", # optional
+                    "SegmentID": "01CA06A0-429B-4EE1-9185-F17D5CE4D0F2",
+                },
+                {
+                    "Tenant":"tenant1",
+                    "Segment": "Segment2",
+                },
+            ],
+            "Tags": [
+                {
+                    "Key": "Platform",
+                    "Value": "Openstack",
+                },
+                {
+                    "Key": "TenantID",
+                    "Value": "D46FD18B-A381-4297-A6D4-9DA594EE71C8",
+                },
+            ], # meta data attached to policies for various external enviorments like openstack/kybernetes
+            "Direction" : "Egress", # can be Egress or Ingress.
+            "Rules": [
+                {
+                    "CidrBlock": "0.0.0.0/0", # IP from L3 header
```

Comment:
If we do that then the 'applied-to' and all concepts of 'src' and 'dst' become tricky, don't they?

I have no problem keeping egress and ingress at the top level.
I also had no problem with CIDR outside of rules. My point was that the peer can be defined in other ways than just CIDR. Peer could be one of several different things.

So, the suggestion was to have an "AppliedTo" and a "Peer" at the top level (don't know if Peer is the best word, but let's just keep it for the moment). AppliedTo is always some tenant/segment combo, while "peer" could be a number of different things: Tenant/segment or CIDR or whatever. 

Then you give a direction so you know if you are talking about ingress (Peer -> AppliedTo) traffic or egress (AppliedTo -> Peer) traffic.

The two peers and the direction are an edge of a directed graph. Seems to make sense to have them live at roughly the same level.

Then you have a rule set, which tells you what specific classification/filtering/whatever you wish to do on that edge of the graph. Seems to make sense to have those rules in their own level.